src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, String name, int radius) {
+        super(color, name = "Circle");
```

Comment:
In the constructor, the assignment `name = "Circle"` is unnecessary since `name` is being set through the `super` call. You should remove `name = "Circle"` and just call `super(color, "Circle")`.